_harp/academy/_faq.md

```diff
@@ -22,7 +22,10 @@ Courses are peer-led and project-based with sixteen students working in four tea
 Primarily, we cover *test-driven* development, using *JavaScript* and *Node.js*. We also cover aspects of UX design, product management and entrepreneurship.
 
 **What projects will I work on in the second part of the programme?**  
-We aim to give you an opportunity to work on a variety of open source, charitable, social enterprise, startup and commercial projects. The programme is under continuous development and the balance between learning and earning a living is always under review.  
+We aim to give you an opportunity to work on a variety of open source, charitable and social enterprise projects. The programme is under continuous development and the balance between learning and creating useful software is always under review.  
+
+**Will I be able to earn a living working on projects?**  
```

Comment:
Will we? Is it not up to the freelancers to bring in their own projects?
Naturally we will pass on anything coming through our doors that is an MVP, but we made the decision a while ago to move away from taking _responsibility_ for this (which is different to helping where we can).